bee-network/bee-autopeering/Cargo.toml

```diff
@@ -19,7 +19,7 @@ bytes = { version = "1.0", default-features = false }
 hash32 = { version = "0.2.1", default-features = false }
 hex = { version = "0.4.3", default-features = false }
 iota-crypto = { version = "0.9.1", default-features = false, features = [ "ed25519", "random", "sha" ] }
-libp2p-core = { version = "0.29.0", default-features = false }
+libp2p-core = { version = "0.30.0", default-features = false }
```

Comment:
Should the autopeering crate also be bumped then ?

Reply:
oh, of course

Reply:
Done